modules/tts/tts-adapters/tts-adapter-cereproc/src/main/java/org/daisy/pipeline/tts/cereproc/impl/CereProcService.java

```diff
@@ -103,7 +104,7 @@ protected CereProcEngine newEngine(
 					logger.warn(prop + " property not set. Defaulting to " + port);
 				}
 			}
-			return new CereProcEngine(CereProcEngine.Variant.STANDARD, this, server, port, client, priority);
+			return new CereProcEngine(CereProcEngine.Variant.STANDARD, this, server, port, client, priority, this.getSSMLxslTransformerURL());
```

Comment:
Hmm.. im not sure its just that simple to override the function, That causes a NullPointer when i tried it. 
I will have to look deeper in to what exactly causes the error.

Reply:
There shouldn't be a NullPointerException because I did this change: https://github.com/daisy/pipeline-modules/commit/4300e0c2cd0cb44d01630b88c70909625fd53f9c. If you get an error please look into it and find out what I did wrong.

Reply:
Ah! i figured it out. it was my own mistake.